src/util/environment.rs

```diff
@@ -0,0 +1,141 @@
+use std::env;
```

Comment:
lets keep this in the build module, rather than util (and the file needs the copyright header)